### PR TITLE
New service methods for the Bluesound component

### PR DESCRIPTION
**Pull request by thrawnarn**
**Description:**
New service methods

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#11450

## Checklist:

  - [ ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`. 
  - [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/media_player.bluesound.markdown [hunk 1]
```diff
@@ -46,3 +46,37 @@ media_player:
...
+
+Remove one or more speakers from a group of speakers. If no `entity_id` is provided, all speakers are unjoined.
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id` | yes | String or list of `entity_id`s that will be separated from their master speaker.
+
+### {% linkable_title Service `bluesound_set_sleep_timer` %}
+
+Sets a timer that will turn off the speaker. For each time you call this it will increase the time by on step. The steps are (in minutes): 15, 30, 45, 60, 90, 0.
```
Comment (frenck) [inline source/_components/media_player.bluesound.markdown]: - `on` -> `one`

File: source/_components/media_player.bluesound.markdown [hunk 2]
```diff
@@ -46,3 +46,37 @@ media_player:
...
+If you increse an ongoing timer of for example 13 minutes, it will increase it to 15. If the timer is set to 90 it will remove the time (hence the 0).
```
Comment (frenck) [inline source/_components/media_player.bluesound.markdown]: - `increse` -> `increase``
- Missing comma: `to 90 it will` -> `to 90, it will`

Comment (thrawnarn) [reply source/_components/media_player.bluesound.markdown]: Thank you, all three is changed

**Comment by thrawnarn** [comment]
This change is actual when this PR is merged.
https://github.com/home-assistant/home-assistant/pull/11450

**Comment by frenck** [comment]
Thank you @thrawnarn for updating!
Approved changes, changed base to `next`.

Can be merged as soon as the parent PR gets merged.

**Pull request merged**